sbndqm/dqmAnalysis/CRT/BernCRTdqm_module.cc

```diff
@@ -88,7 +89,7 @@ class sbndaq::BernCRTdqm : public art::EDAnalyzer {
 
    uint64_t lastbighit[32];
 
-  bool debug = true;
+  bool debug = false;
```

Comment:
Since no code is trying to change it:
```suggestion
  bool const debug = false;
```
(also answers a reader question "how to activate those debugging messages").